org.lflang/src/org/lflang/formatting2/LFFormatter.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * generated by Xtext 2.26.0
+ */
+
+package org.lflang.formatting2;
+
+import java.util.List;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.xtext.formatting2.FormatterRequest;
+import org.eclipse.xtext.formatting2.IFormatter2;
+import org.eclipse.xtext.formatting2.regionaccess.ITextReplacement;
+import org.eclipse.xtext.formatting2.regionaccess.ITextSegment;
+import org.eclipse.xtext.formatting2.regionaccess.internal.TextReplacement;
+import org.eclipse.xtext.util.CancelIndicator;
+import org.eclipse.xtext.validation.CheckMode;
+import org.eclipse.xtext.validation.IResourceValidator;
+
+import org.lflang.ast.FormattingUtils;
+
+import com.google.inject.Inject;
+
+public class LFFormatter implements IFormatter2 {
+
+    @Inject
+    private IResourceValidator validator;
+
+    @Override
+    public List<ITextReplacement> format(FormatterRequest request) {
+        // TODO: Use a CancelIndicator that actually cancels?
```

Comment:
Could you please create an issue to track this `TODO`?

Reply:
Done (#1342).